### PR TITLE
csv docs

### PR DESCRIPTION
**Pull request by RozziP**

### CODE REVIEW

File: docs/workflows/logic/csv-decode.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+flowclass:
+  - Application
+  - Experience
+  - Edge 1.6.0
+description: Learn more about the Losant CSV Decode Node.
+---
+
+# CSV Decode Node
+
+The CSV Decode Node allows a workflow to decode a CSV String on the payload into an object.
```
Comment (anaptfox) [inline docs/workflows/logic/csv-decode.md]: in all places where you have "CSV String" string shouldn't be capitalized.

File: docs/workflows/logic/csv-decode.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{
+...
+  "data": {
+    "csvString": "Food,Drink\nBurger,Soda\nTaco,Water\n"
+  }
+...
+}
+```
+
+the CSV Decode node would give the following output payload.
```
Comment (anaptfox) [inline docs/workflows/logic/csv-decode.md]: "the CSV Decode node would give the following output payload."

should be: 

the CSV Decode Node would give the following output payload:

File: docs/workflows/logic/csv-decode.md (hunk 1 above)
Comment (eeerin) [inline docs/workflows/logic/csv-decode.md]: an array of objects.

File: docs/workflows/logic/csv-decode.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+This required field specifies the payload path where the CSV String to be decoded is located.
+
+![CSV Decode Node Input](/images/workflows/logic/csv-decode-node-input.png "CSV Decode Node Input")
+
+### Format Configuration
+
+![CSV Decode Node Config](/images/workflows/logic/csv-decode-node-config.png "CSV Decode Node Config")
+
+This section contains four fields that specify the format of the incoming CSV String:
```
Comment (eeerin) [inline docs/workflows/logic/csv-decode.md]: . instead of :

File: docs/workflows/logic/csv-decode.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+* **Record Delimiter:** (Optional) This is the character used to separate rows of data.
```
Comment (anaptfox) [inline docs/workflows/logic/csv-decode.md]: All items in this list should end in a period.

File: docs/workflows/logic/csv-decode.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Result Path
+
+This required field specifies the payload path where the decoded object array will be placed.
+
+![CSV Decode Node Result](/images/workflows/logic/csv-decode-node-result.png "CSV Decode Node Result")
+
+## Output
+
+In this particular instance, the workflow will decode the CSV string at the `data.csvString` path and place the resulting array of objects at `data.out`
```
Comment (anaptfox) [inline docs/workflows/logic/csv-decode.md]: missing period

File: docs/workflows/logic/csv-decode.md (hunk 2 above)
Comment (eeerin) [reply docs/workflows/logic/csv-decode.md]: capitalize the 't' in the.

File: docs/workflows/logic/csv-decode.md (hunk 3 above)
Comment (eeerin) [reply docs/workflows/logic/csv-decode.md]: ignore this comment if this is how @anaptfox wants it:

File: docs/workflows/logic/csv-encode.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+* **Payload Path:** Specify a payload path that contains an array of headers to include. Any object keys not included in the array will be excluded from the output.
+
+### Format Configuration
+
+![CSV Encode Node Config](/images/workflows/logic/csv-encode-node-config.png "CSV Encode Node Config")
+
+This section contains four fields that specify the format for the encoded CSV string:
+
+* **Record Delimiter:** (Optional) This is the character used to separate rows of data.
```
Comment (eeerin) [inline docs/workflows/logic/csv-encode.md]: This isn't really optional on the front end like it is on the backend. And by default, it's going to be \n.

Comment (eeerin) [reply docs/workflows/logic/csv-encode.md]: same for encode.

File: docs/workflows/logic/csv-encode.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+* **Delimiter:** (Optional) A [string template](/workflows/accessing-payload-data/#string-templates) to override the delimiter. This character is used to separate columns of data. Must be a single character. Leave blank to default to comma (,)
```
Comment (anaptfox) [inline docs/workflows/logic/csv-encode.md]: some periods in the list items here

File: docs/workflows/logic/csv-encode.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+     "Drink": "Water",
+     "Food": "Taco"
+    }
+   ]
+  }
+...
+}
+```
+
+the CSV Encode node would give the following output payload.
```
Comment (anaptfox) [inline docs/workflows/logic/csv-encode.md]: Node

File: docs/workflows/logic/csv-encode.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+### Result Path
+
+This required field specifies the payload path where the encoded CSV string will be placed.
+
+![CSV Encode Node Result](/images/workflows/logic/csv-encode-node-result.png "CSV Encode Node Result")
+
+## Output
+
+In this particular instance, the workflow will encode the object array at the `data.in` path and place the resulting CSV string at `data.csvString`
```
Comment (anaptfox) [inline docs/workflows/logic/csv-encode.md]: Period

Comment (eeerin) [inline docs/workflows/logic/csv-encode.md]: .

File: docs/workflows/logic/csv-encode.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+This section contains four fields that specify the format for the encoded CSV string:
+
+* **Record Delimiter:** (Optional) This is the character used to separate rows of data.
+
+* **Delimiter:** (Optional) A [string template](/workflows/accessing-payload-data/#string-templates) to override the delimiter. This character is used to separate columns of data. Must be a single character. Leave blank to default to comma (,)
+
+* **Quote:** (Optional) A string template to override the quote character. Must be a single character. Leave blank to default to double-quote (")
+
+* **Escape:** (Optional) A string template to override the escape character. Must be a single character. Leave blank to default to double-quote (")
```
Comment (eeerin) [inline docs/workflows/logic/csv-encode.md]: delimiter, quote and escape all need `.`'s.

File: docs/workflows/logic/csv-encode.md (hunk 6 above)
Comment (RozziP) [reply docs/workflows/logic/csv-encode.md]: I wasn't sure if that needed to be said, since it's a select and the user will see that it's on \n by default

**Pull request merged**